CONTRIBUTING.md

```diff
@@ -4,13 +4,33 @@ This is a [Jazzband](https://jazzband.co/) project. By contributing you agree
 to abide by the [Contributor Code of Conduct](https://jazzband.co/about/conduct)
 and follow the [guidelines](https://jazzband.co/about/guidelines).
 
+## Getting started
+
+Install tox:
+
+```console
+$ python -m pip install tox --user
+```
+
+Create a virtual environment, with `pip-tools` in development mode and its test dependencies,
+using your current python version:
+
+```console
+$ tox --devenv .venv -e py
+```
+
+Activate the virtual environment:
+
+```console
+$ source .venv/bin/activate
```

Comment:
One thing I'm concerned about is that it doesn't make it clear that running tests through tox will not use this env, it can be used for development detached from tox and if one wants to run pytest there, they should do so directly.